docs/content/0.1.0/docs/dj-concepts/node-dependencies.rst

```diff
@@ -1,3 +1,53 @@
 ---
 weight: 40
----
\ No newline at end of file
+---
+
+-----------------
+Node Dependencies
+-----------------
+
+Relationships between nodes are tracked by a DJ server. A node's position in the DJ DAG is determined by the node's definition,
+particularly the query. Node queries reference other DJ nodes as tables and this is what defines upstream and downstream
```

Comment:
Should we remove the "as tables" part of "reference other DJ nodes as tables"? I'm coming at this from the angle of it potentially being confusing to talk about nodes and tables as if they were similar, when they're actually different things in DJ.

Reply:
Good call, updated!